examples/react-apps/src/pages/metamask-connection/__tests__/connect-wallet.test.tsx

```diff
@@ -113,4 +115,38 @@ describe("Connect wallet", () => {
     await screen.findByText(/chain id: 0x3/i);
     expect(screen.getByText(/balance: 5.00/i)).toBeInTheDocument();
   });
+  test("User should be able to sign a transaction", async () => {
+    const bobsWallet = ethers.Wallet.createRandom();
+    // Start with a connected bobsWallet
+    testingUtils.mockRequestAccounts([bobsWallet.address]);
```

Comment:
This `mockRequestAccounts` should be safe to be deleted as the `mockConnectedWallet` is used :)

Reply:
Interesting, I was having problems with this in my own commit, will remove here.